org.scintilla.SciTE.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.scintilla.SciTE
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: SciTE
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=host
+  
+modules:
+
+- name: SciTE
+  buildsystem: simple
+  sources:
+  - type: file
+    url: https://www.scintilla.org/scite446.tgz
+    sha256: 1c2e6eee6c8aa2e52983d9713a4b1c97d06c376324a3ec8932adfcb9e55d16d1
+    dest-filename: scite.tgz
+  build-commands:
+  - tar -zxf scite.tgz
```

Comment:
You shouldn't have to do that. Use the type `archive` for the source, with a `strip-components` set to 0, and let flatpak handle it.